CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Author: Jingting Liu
+# CMakeLists for Mavlink library
+cmake_minimum_required(VERSION 3.2.0)
+
+PROJECT(Mavlink2)
+
+include_directories(${CMAKE_CURRENT_LIST_DIR})
```

Comment:
For someone trying to use Mavlink, you'd need to give it to them as a library. See ZeroPilot-SW/AutoPilot/DataStructures/CmakeLists.txt for how to make a cmakelists that produces a library.

For your testing section, I'd suggest you just have the whole thing commented out.